packages/pyolite-kernel-extension/src/index.ts

```diff
@@ -12,7 +12,7 @@ import { PyoliteKernel } from '@jupyterlite/pyolite-kernel';
 /**
  * The default CDN fallback for Pyodide
  */
-const PYODIDE_CDN_URL = 'https://cdn.jsdelivr.net/pyodide/v0.17.0/full/pyodide.js';
+const PYODIDE_CDN_URL = 'https://cdn.jsdelivr.net/pyodide/dev/full/pyodide.js';
```

Comment:
If there was a way to pin on a commit / hash, maybe we could already start shipping it.

Reply:
ha, would rather not, just yet... i think waiting for the npm releases will be worth it...